volume/drivers/fake/fake.go

```diff
@@ -85,12 +87,14 @@ func newFakeDriver(params map[string]string) (*driver, error) {
 		return nil, err
 	}
 	inst := &driver{
-		IODriver:           volume.IONotSupported,
-		StoreEnumerator:    common.NewDefaultStoreEnumerator(Name, kv),
-		StatsDriver:        volume.StatsNotSupported,
-		QuiesceDriver:      volume.QuiesceNotSupported,
-		CloudMigrateDriver: volume.CloudMigrateNotSupported,
-		kv:                 kv,
+		IODriver:              volume.IONotSupported,
+		StoreEnumerator:       common.NewDefaultStoreEnumerator(Name, kv),
+		StatsDriver:           volume.StatsNotSupported,
+		QuiesceDriver:         volume.QuiesceNotSupported,
+		CloudMigrateDriver:    volume.CloudMigrateNotSupported,
+		FilesystemTrimDriver:  volume.FilesystemTrimNotSupported,
```

Comment:
Please add a Portworx Jira and in OpenStorage github issue to do this implementation in the future. No need to link the Jira issue here, just the Github issue.

Reply:
Thanks for reviewing the code and helping me understand the libopenstorage framework for updates.